src/design-system/docs/package.json

```diff
@@ -17,8 +17,8 @@
   },
   "dependencies": {
     "@capsizecss/vanilla-extract": "1.0.0",
-    "@vanilla-extract/css": "1.6.3",
-    "@vanilla-extract/sprinkles": "1.3.1",
+    "@vanilla-extract/css": "1.9.0",
```

Comment:
This version was bumped to fix a bug with CSS variable names containing spaces, which we tripped over due to our new ` (Deprecated)` suffix.